src/store/domains/globalSearch.ts

```diff
@@ -69,7 +74,8 @@ export default class GlobalSearch implements GlobalSearchStoreInterface {
   }
 
   setSearchResults(results: GlobalSearchResults) {
-    this.results = results;
+    this.results = results.items;
```

Comment:
Hier haben wir leider ein Type-Problem.
`GlobalSearchResults` beschreibt nämlich ein Array mit Elementen vom Typ "GlobalSearchArtifact".
Hier wird aber davon ausgegangen, dass `results` ein Objekt mit den Eigenschaften `items` und `meta` ist.
Also die Struktur aufweist, die `assembleResultData` in Form des Return-Values vorgibt.